graph-platform/neo4j-desktop/neo4j-desktop.adoc

```diff
@@ -44,21 +44,28 @@ Step 2 on the download page instructions shows you how to set up your first proj
 [#desktop-activate]
 == Activation
 
-image::{img}desktop_registration_dec2019.jpg[link="{img}desktop_registration_dec2019.jpg",float="right"]
+[#desktop-first-time]
+== Opening Neo4j Desktop for the First Time
 
 If you are opening Neo4j Desktop for the first time, it should ask you to register the software with an activation key.
-When you downloaded Neo4j Desktop, your unique activation key was displayed at the top of the instructions page.
+This activation key is generated when you first download Neo4j Desktop and will be displayed on the download confimration page.
 While we can copy it to the clipboard and paste it into the provided input in Neo4j Desktop, we also recommend copying it to a safe place as a backup (local file, password vault, or another method).
 
+image::{img}desktop_registration_may2020.jpg[link="{img}desktop_registration_may2020.jpg",float="right"]
+
+Copy this activation key into the Software regisration form and click Activate to continue.
+
+As a new user to Neo4j Desktop, you will be greated with a new sample database containing https://sandbox.neo4j.com/?usecase=recommendations[Movies and Actors].
+
+image::{img}first_graph_experience.png[link="{img}first_graph_experience.png",float="right"]
+
+Clicking the Browser Guide link for the Sample database will open up the Neo4j Browser with a Browser Guide that will guide you through your first queries on this Neo4j database.
+
 [#desktop-navigate]
 == Navigation
 
 The menu along the left side of Neo4j Desktop shows high-level icons for projects, graph applications, settings, activation keys, and Neo4j info.
 At the top, the folder with a bookmark (image:{img}projects_icon.jpg[width=20]) shows the list of projects you have created.
```

Comment:
I wonder if these icons can be found somewhere in fontawesome. But that's not critical :) for the publication. Just curiousity.

Reply:
Good question, I assume they are Semantic UI.  I can look at including these during the redesign